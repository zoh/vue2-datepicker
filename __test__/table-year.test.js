import { mount } from '@vue/test-utils';
import TableYear from '../src/calendar/table-year.vue';

let wrapper;

afterEach(() => {
  wrapper.destroy();
});

describe('TableYear', () => {
  it('decade=2010', () => {
    wrapper = mount(TableYear, {
      propsData: {
        decade: 2010,
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
