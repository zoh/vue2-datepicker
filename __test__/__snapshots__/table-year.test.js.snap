// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableYear decade=2010 1`] = `
<table
  class="mx-table mx-table-year"
>
  <tr>
    <td
      class="cell"
      data-year="2010"
    >
      <div>
        2010
      </div>
    </td>
    <td
      class="cell"
      data-year="2011"
    >
      <div>
        2011
      </div>
    </td>
  </tr>
  <tr>
    <td
      class="cell"
      data-year="2012"
    >
      <div>
        2012
      </div>
    </td>
    <td
      class="cell"
      data-year="2013"
    >
      <div>
        2013
      </div>
    </td>
  </tr>
  <tr>
    <td
      class="cell"
      data-year="2014"
    >
      <div>
        2014
      </div>
    </td>
    <td
      class="cell"
      data-year="2015"
    >
      <div>
        2015
      </div>
    </td>
  </tr>
  <tr>
    <td
      class="cell"
      data-year="2016"
    >
      <div>
        2016
      </div>
    </td>
    <td
      class="cell"
      data-year="2017"
    >
      <div>
        2017
      </div>
    </td>
  </tr>
  <tr>
    <td
      class="cell"
      data-year="2018"
    >
      <div>
        2018
      </div>
    </td>
    <td
      class="cell"
      data-year="2019"
    >
      <div>
        2019
      </div>
    </td>
  </tr>
</table>
`;
